clients/src/main/java/org/apache/kafka/clients/producer/internals/BufferPool.java

```diff
@@ -43,6 +43,8 @@
  */
 public class BufferPool {
 
+    static final String BUFFERPOOL_WAIT_TIME_SENSOR_NAME = "bufferpool-wait-time";
```

Comment:
We don't need the BUFFERPOOL prefix here since it's already defined in the `BufferPool` class. Thanks for the PR, will hopefully review it soon.